src/components/LinkItem/index.js

```diff
@@ -7,7 +7,7 @@ const ROW_BREAKPOINT = 600;
 
 const getDate = (link) => {
     if (!(link?.createdAt?.toDate && link.createdAt.toDate instanceof Function)) {
-        return new Date(0);
+        return new Date();
```

Comment:
The goal here is to show an invalid date. That's why we need `Date(0)`.

Reply:
Fixed on commit [dac7658](https://github.com/agenciagift/feed/pull/6/commits/dac7658c14b8ba97b1c4d75eb469cc64e4671edf)